racetrack_commons/racetrack_commons/deploy/job_type.py

```diff
@@ -45,7 +48,7 @@ def gather_job_types(
     job_types: dict[str, JobType] = {}
     job_family_versions: dict[str, list[JobType]] = defaultdict(list)
 
-    plugin_results: list[dict[str, tuple[Path, Path]]] = plugin_engine.invoke_plugin_hook(PluginCore.fatman_job_types)
+    plugin_results: list[dict[str, JobTypeImagePaths]] = plugin_engine.invoke_plugin_hook(PluginCore.fatman_job_types)
```

Comment:
Are you sure it's correct change? it's replacing `tuple[Path, Path]` with `list[tuple[Path, Path]]`

Reply:
Well, actually it should be `list[tuple[Path, Path]]`, but in the next lines I tried to make it backwards-compatible and also work if `tuple[Path, Path]` was given.

Do you think it should be more strict? (then I'll need to update all the existing job type plugins)

Reply:
Or maybe I should use Union type: `list[tuple[Path, Path]] | tuple[Path, Path]`

Reply:
Ha, wouldn't have known it could accept both. So yeah, it's good to declare it like that explicitely.